gradient_utils/metrics.py

```diff
@@ -56,7 +56,7 @@ def get_workload_id():
     return _get_experiment_id()
 
 
-def add_metrics(
+def _add_metrics(
```

Comment:
Removing from init only prevents people from importing *. To make it explicit unsupported at the moment, this makes the method private.